api/krusty/component_test.go

```diff
@@ -33,7 +33,7 @@ func writeK(path string, content string) FileGen {
 }
 
 func writeTestBase(th kusttest_test.Harness) {
-	th.WriteK("/app/base", `
+	th.WriteK("/base", `
```

Comment:
Is there some reason here that these all have to be absolute paths?

It's preferable to omit the `/`, so readers don't copy this and think that kustomization directories aren't relocatable.

Reply:
ah sorry, I meant to edit this earlier and got distracted by something else. Thanks for merging; I can make a separate PR for this